### DISCUSSION TITLE
How to remove these borders when running a terminal (kitty)?

### DISCUSSION THREAD

**Discussion by qvc0** [i3/i3#5841] [Q&A]
![2023-12-27_21-16](https://github.com/i3/i3/assets/142407322/4a46ff29-c1bc-4252-933c-916bce275d8c)
When i launch kitty these borders appear and i dont know how to remove them. They appear only when you are running terminal in non-fullscreen mode.

**Comment by orestisfl** [comment]
These appear to be drawn by kitty, check the documentation of that application, perhaps https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.hide_window_decorations
